sdk/samples/iot/paho_iot_pnp_component_sample.c

```diff
@@ -460,7 +460,7 @@ static void process_device_property_message(
     az_span property_message_span,
     az_iot_hub_client_properties_message_type message_type)
 {
-  az_result rc = az_iot_hub_client_properties_update_get_publish_topic(
+  az_result rc = az_iot_hub_client_properties_get_reported_publish_topic(
```

Comment:
am I crazy or is this call really needed here? The publish message doesn't seem to be referenced elsewhere in this function

Reply:
Oh - it's even worse than not being used.  It's used in `pnp_thermostat_process_property_update`, when `publish_mqtt_message` uses it as a topic.

So if there's just one property it works but it's confusing as all get-out.
If there's >1 property, it ends up using the same RID.

I'll fix this.